rapyuta_io_sdk_v2/client.py

```diff
@@ -1102,10 +1113,15 @@ def create_revision(
 
     @handle_and_munchify_response
     def put_keys_in_revision(
-        self, name: str, revision_id: str, configValues: list[(object)], **kwargs
+        self, name: str, revision_id: str, configValues: list[(dict)], **kwargs
```

Comment:
Use snake case for all Python objects and variables.
```suggestion
        self, name: str, revision_id: str, config_values: list[(dict)], **kwargs
```

Reply:
@guptadev21 Check in other places also. Make it consistent.